src/platform/cmake/FindFeature.cmake

```diff
@@ -12,9 +12,9 @@ function(find_feature FEATURE_NAME FEATURE_REQUIRES)
 		set(FOUND OFF)
 		foreach(REQUIRE ${NAMELIST})
 			if(NOT ${REQUIRE}_FOUND)
-				find_package(${REQUIRE} QUIET)
+				pkg_search_module(${REQUIRE} QUIET ${REQUIRE})
 				if(NOT ${REQUIRE}_FOUND)
-					pkg_search_module(${REQUIRE} ${REQUIRE})
+					find_package(${REQUIRE})
```

Comment:
I cannot condone this change. The issue appears to be upstream (I filed a bug, but I'm unsure what to actually do about it...CMake is bad that way), and this is an actual behavior change from intended. Please build with libzip disabled in the meantime.